packages/band-arithmetic/band-arithmetic.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const _ = require('underscore');
+const parse_georaster = require("georaster");
+
+const get = require('../get/get');
+const utils = require('../utils/utils');
+const convert_geometry = require('../convert-geometry/convert-geometry');
+const intersect_polygon = require('../intersect-polygon/intersect-polygon');
+
+const logger = require('../../logger');
+const parse = require('mathjs').parse;
+
+const regex_multi_character = /[A-z]{2}/g;
+
+const is_leaf_node = node => !node.op;
+
+const variables = [...Array(13)].map((val, i) => String.fromCharCode(i + 65).toLowerCase());
+const operations = {
+  add: (a, b) => a + b,
+  subtract: (a, b) => a - b,
+  multiply: (a, b) => a * b,
+  divide: (a, b) => a / b
+};
+
+const get_value = (input, band_values) => {
+  if (input.value) return input.value;
+
+  const variable_index = variables.findIndex(variable => variable === input.name);
+  return band_values[variable_index];
+};
+
+const compute_value = (node, band_values) => {
+  const left_node = node.args[0];
+  const right_node = node.args[1];
+
+  const operation = node.fn;
+
+  let left_value;
+  if (left_node.content) { // if the node represents parentheses, it will be an object with a single property "content" which contains a node
+    left_value = compute_value(left_node.content, band_values);
+  } else if (is_leaf_node(left_node)) {
+    left_value = get_value(left_node, band_values);
+  } else {
+    left_value = compute_value(left_node, band_values);
+  }
+
+  let right_value;
+  if (right_node.content) { // if the node represents parentheses, it will be an object with a single property "content" which contains a node
+    right_value = compute_value(right_node.content, band_values);
+  } else if (is_leaf_node(right_node)) {
+    right_value = get_value(right_node, band_values);
+  } else {
+    right_value = compute_value(right_node, band_values);
+  }
+
+  return operations[operation](left_value, right_value);
+};
+
+const get_band_rows = (bands, index) => {
```

Comment:
not sure you see it bc I have the Code Climate extension installed, but get_band_rows and get_band_values could probably be consolidated into one flatten function?  I'll still approve tho